Big-Spender/readme.md

```diff
@@ -67,42 +68,55 @@ INSERT YOUR QUERY HERE
 
 **You:** Then here's the query for that:
 
-```sql
+````sql
 INSERT YOUR QUERY HERE
-```
+```   SELECT * FROM spends WHERE LOWER(description) LIKE '%fee%' OR LOWER(description) LIKE '%fees%'```
+````
 
 **Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
-```sql
+````sql
 INSERT YOUR QUERY HERE
-```
+```SELECT * FROM spends WHERE expense_area_id =(SELECT expense_area_id FROM expense_areas WHERE expense_area = 'Better Hospital Food') ```
```

Comment:
I don't think you need the query nesting here. You can add the where clause directly.

i.e. SELECT * FROM spends WHERE expense_area = ''